File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Declare variables
    int n = 0;

    // Get input from the user
    while (n < 1 || n > 8)
    {
        n = get_int("Height: ");
    }

    // To create pyrimaid
    // For each row
    for (int i = 1; i < n + 1; i++)
    {
        int space = (n - i);
        int hash = i;

        // Forward pyrimaid (space)
        for (int j = space; j != 0; j--)
        {
            printf(" ");
        }
        // Forward pyrimaid (hash)
        for (int j = hash; j != 0; j--)
        {
            printf("#");
        }
        // Middle
        printf("  ");
        // Backward pyrimaid (hash)
        for (int j = hash; j != 0; j--)
        {
            printf("#");
        }
        // Go to new line
        printf("\n");
    }
}

File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int

height = None
while (not isinstance(height, int) or (height < 1 or height > 8)):
    try:
        height = int(get_int("Height: "))
    except:
        print("Height should between 1 and 8.")

# create each row
for i in range(height):

    # create whitespace
    for _ in range(height-1 - i):
        print(" ", end="")

    # create '#'
    for _ in range(i+1):
        print("#", end="")

    # go to new line
    print("\n", end="")
